README.md

```diff
@@ -13,19 +13,17 @@ This module requires the following modules/libraries:
 * [Islandora](https://github.com/islandora/islandora)
 * [Tuque](https://github.com/islandora/tuque)
 * [Libraries API](https://drupal.org/project/libraries)
-* [JW Player](www.longtailvideo.com/download/jwplayer-free-6-1-2972.zip)
+* [JW Player version 6](https://github.com/jwplayer/jwplayer/releases/tag/v6.12.0)
 
 ## Installation
 
 Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
 
-Install the JW Player available [here](http://www.longtailvideo.com/jw-player/download/) to your `sites/all/libraries` folder.
-
-Note: You must register for an account with JWPlayer to download the library.
+Install the JW Player available [here](https://github.com/jwplayer/jwplayer/releases/tag/v6.12.0) to your `sites/all/libraries` folder.  You will have to build it yourself.
```

Comment:
Do we have any idea how to do this? Even a starting place might reduce the amount of support requests.